game/game/playercontrol.cpp

```diff
@@ -196,12 +198,13 @@ void PlayerControl::tickFocus() {
   currentFocus = findFocus(&currentFocus);
   setTarget(currentFocus.npc);
 
-  if(!ctrl[Action::ActionGeneric])
+  if(!ctrl[Action::ActionGeneric] || Gothic::inst().world()->player()->interactive()!=nullptr)
```

Comment:
1. nullptr checks - there is no guaranies , that player always will be around
2. What is the purpose if this change?

Reply:
Again for G1 controls. Some ladders showed buggy behavior when interacting at top. This change prevents interaction, if player already interacts.

Reply:
Should this kind of check then be placed into `Focus World::findFocus(const Npc &pl, const Focus& def)`?
if player already interacts with something, then search can simply early-out.